discovery/src/main/scala/org/apache/pekko/discovery/ServiceDiscovery.scala

```diff
@@ -150,7 +150,7 @@ final class Lookup(val serviceName: String, val portName: Option[String], val pr
   require(serviceName.trim.nonEmpty, "'serviceName' cannot be empty")
 
   /**
-   * Which port for a service e.g. Akka remoting or HTTP.
+   * Which port for a service e.g. Apache Pekko remoting or HTTP.
```

Comment:
Some parts are `Pekko` but some are `Pekko`, it there any guide about this?

Reply:
not really - some times, it seems more correct to use Apache Pekko.

At some point, we'll probably need someone to specifically go through all the docs and decide when Pekko or Apache Pekko is used.